src/utils/resources/template/utils/constants.ts

```diff
@@ -96,7 +96,7 @@ export enum BOOT_SOURCE {
 export const BOOT_SOURCE_LABELS = {
   [BOOT_SOURCE.PVC]: 'PVC',
   [BOOT_SOURCE.DATA_SOURCE]: 'PVC',
-  [BOOT_SOURCE.DATA_SOURCE_AUTO_UPLOAD]: 'PVC (auto upload)',
+  [BOOT_SOURCE.DATA_SOURCE_AUTO_UPLOAD]: 'PVC (auto import)',
```

Comment:
Consider changing key value AUTO_UPLOAD to AUTO_IMPORT

Reply:
nice +1

Reply:
@metalice updated, please re-review